onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -1156,8 +1155,8 @@ Status PoolOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const N
   const auto& op_type = node_unit.OpType();
 
   int32_t op_code;
-  bool is_qlinear_average_pool = op_type == "QLinearAveragePool";
-  bool is_average_pool = op_type == "AveragePool" || is_qlinear_average_pool;
+  bool is_quant_pool = IsQuantizedOp(node_unit);
+  bool is_average_pool = op_type == "AveragePool" || op_type == "QLinearAveragePool";
   if (is_average_pool || op_type == "GlobalAveragePool")
```

Comment:
Does this need to also check 'QDQAveragePool'?

Reply:
actually I am not really sure about this.. 

as I guess my understanding is that there's no `op_type` name really called 'QDQAveragePool' . for the averagepool op in a qdq group,  the op_type name will still show as 'AveragePool'.

Reply:
op_type is the type of the node or the target node of the node_unit